calico_node/Makefile

```diff
@@ -1,5 +1,9 @@
 ###############################################################################
+# vX.Y format: update during major release process
+V_RELEASE_STREAM := v2.6
+RELEASE_STREAM ?= $(V_RELEASE_STREAM)
```

Comment:
These two lines can be collapsed to just `RELEASE_STREAM ?= v2.6`.  (The dance is useful in other cases because of the `$(shell ...)` taking noticeable time; but not in this case.)

Reply:
done.